Zend/tests/gh15938_001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+GH-15938
+--FILE--
+<?php
+
+#[AllowDynamicProperties]
+class C {}
+
+$obj = new C();
+// $obj->a = str_repeat('a', 10);
```

Comment:
Note: This line isn't even necessary to trigger the leak. `$this->a .= ` prepares the index in the property table and stores the pointer in the `VAR` passed to `ASSIGN_OP`. `unset($obj->a)` will then remove the key and NULL value that were just created. The actual store will then occur in a dead zval.